.gitlab-ci.yml

```diff
@@ -729,6 +729,7 @@ library:ci-corn:
   needs:
   - build:edge+flambda
   - plugin:ci-bignums
+  - plugin:ci-elpi
```

Comment:
just depend on elpi_hb, that way we don't need a new elpi-only job

Reply:
Btw.: Makefile.ci already has ci-elpi.

Reply:
the combined job is below 2min https://gitlab.inria.fr/coq/coq/-/jobs/4948041
so the overhead of having a separate job (start a doker, download artifacts, etc) is not worth it

Reply:
But in the Makefile I should still depend on elpi only?

Reply:
>But in the Makefile I should still depend on elpi only?


yes

Reply:
I updated the PR as discussed: in the yaml file CoRN depends on elpi_hb, in the makefile on elpi.